.github/file-filters.yml

```diff
@@ -0,0 +1,4 @@
+# This is used by the action https://github.com/dorny/paths-filter
+
+high_risk_code: &high_risk_code
+  - ".github/file-filters.yml"
```

Comment:
cool! 

is this how we reference file paths?

```yml
high_risk_code: &high_risk_code
  - "dart/lib/src/risky_file.dart"  
  - "flutter/lib/src/risky_file.dart"
```

Reply:
yep, just full repo paths, you can check https://github.com/getsentry/sentry-cocoa/blob/main/.github/file-filters.yml for inspiration

Reply:
Okay, I'll think about the code paths that would benefit from this

Reply:
What do we define as risky? Is this strictly defined as code paths that are prone to crashing?

Reply:
the way to think about this is - what can break the users in significant ways which is not currently automatically ensured/validated/tested/etc. in a different way - also try to strike a good balance between missing things and over-reporting (I wouldn't expect this warning to need to be shown in every other PR, for example)